mgmtfn/dockplugin/netDriver.go

```diff
@@ -433,14 +435,27 @@ func GetDockerNetworkName(nwID string) (string, string, string, error) {
 			var tenantName, netName, serviceName string
 			names := strings.Split(nw.Name, ".")
 			if len(names) == 2 {
-				// has only network.tenant format
-				tenantName = names[1]
-				netName = names[0]
+				// determine if this is service.network on default tenant or network.tenant
```

Comment:
should we use 'service.network/tenant' to avoid this confusion, then 'service.network' is valid and 'network/tenant' is valid and 'service.network/tenant' is valid too. This is assuming that '/' is allowed in the list as well.